django/db/models/sql/query.py

```diff
@@ -162,6 +162,7 @@ def __init__(self, model, where=WhereNode):
         self.select_for_update = False
         self.select_for_update_nowait = False
         self.select_for_update_skip_locked = False
+        self.select_for_update_of = {}
```

Comment:
I think that `self.select_for_update_of` should be a tuple rather than a dictionary.